client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -103,6 +104,20 @@ export class SchemaEditorFiles {
       this.dropzoneOptions
     );
 
+    this.dropzone.on("sending", (file, xhr, data) => {
+      if (
+        this.options.pathPrefix !== null &&
+        this.options.pathPrefix !== undefined
+      ) {
+        if (file.fullPath) {
+          data.append(
+            "fullPath",
+            `${this.options.pathPrefix}/${this.uuid}/${file.fullPath}`
```

Comment:
Lets rename the property `pathPrefix` to `keyPrefix`. Q-Server already has an option called `keyPrefix` see here -> https://github.com/nzzdev/Q-server/blob/dev/plugins/file/index.js#L88. I would like to name it the same way.